ImageIO/Pillow unexpected keyword "pilmode"

```
self = <imageio.plugins.pillow.JPEGFormat.Reader object at 0x7f9796f86630>
format = <Format JPEG-PIL - JPEG (ISO 10918)>
request = <imageio.core.request.Request object at 0x7f9796f865c0>
    def __init__(self, format, request):
        self.__closed = False
        self._BaseReaderWriter_last_index = -1
        self._format = format
        self._request = request
        # Open the reader/writer
>       self._open(**self.request.kwargs.copy())
E       TypeError: _open() got an unexpected keyword argument 'pilmode'
```

See #330

--- a/requirements.txt
+++ b/requirements.txt
@@ -9,4 +9,3 @@ scipy>=0.13.3
 imageio>=2.3.0
 six
 tqdm
-contextlib2

--- a/requirements.txt
+++ b/requirements.txt
@@ -6,7 +6,7 @@ pandas>=0.20.0
 pillow
 requests
 scipy>=0.13.3
-imageio
+imageio>=2.3.0
 six
 tqdm
 contextlib2

